guides/source/active_record_querying.md

```diff
@@ -50,7 +50,7 @@ class Role < ApplicationRecord
 end
 ```
 
-Active Record will perform queries on the database for you and is compatible with most database systems (MySQL, PostgreSQL and SQLite to name a few). Regardless of which database system you're using, the Active Record method format will always be the same.
+Active Record will perform queries on the database for you and is compatible with most database systems (MySQL, MariaDB, PostgreSQL and SQLite to name a few). Regardless of which database system you're using, the Active Record method format will always be the same.
```

Comment:
…is compatible with most database systems, including MySQL, MariaDB, PostgreSQL, and SQLite.